pypulseq/make_extended_trapezoid_area.py

```diff
@@ -13,20 +13,18 @@
 def make_extended_trapezoid_area(
     area: float, channel: str, grad_end: float, grad_start: float, system: Opts
```

Comment:
While we're at it, change system to be optional with default `system = None`, and initialize with `system = Opts()` below (see other functions for an example).